gooddata-sdk/gooddata_sdk/catalog/workspace/declarative_model/workspace/logical_model/ldm.py

```diff
@@ -34,6 +34,65 @@ def store_to_disk(self, workspace_folder: Path) -> None:
         if self.ldm is not None:
```

Comment:
Why are docstrings for 

- modify_mapped_data_source
- change_tables_columns_case

Placed above methods?

Reply:
Fixed